### PR TITLE
Install selfoss on a subfolder

### PR DESCRIPTION
**Pull request by Binnette**
Hello,

I "somehow" manage to make selfoss work on a subfolder of my nohost domain (instead of a custom subdomain).
I'm really not an expert in YunoHost nor Nginx. So please review my changes. For exemple my "solution" do not show the YunoHost button in the bottom right corner of the browser;

For the Nginx.conf I just adapted it from [selfos documentaiton](https://github.com/fossar/selfoss/wiki/nginx-configuration)

image

NB: you can test my package by installing it through my repo: https://github.com/Binnette/selfoss_ynh

### CODE REVIEW

**Comment by ericgaspar** [comment]
@Binnette Thanks! I have tested selfoss in subfolder and it works but it doesnt work in a subdomain... Ideally the app should work on subdomain and subfolder.

**Comment by farvardin** [comment]
@Binnette c'est parfait comme ça ! Je suis heureux de retrouver mon selfoss, et c'est mieux que rien si ça fonctionne seulement en sous dossier et non pas en sous domaine...

File: conf/nginx.conf [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+#sub_path_only rewrite ^__PATH__$ __PATH__/ permanent;
+location __PATH__ {
 
-  # Path to source
-  alias __FINALPATH__/;
-
-  index index.php;
-
-  try_files $uri /public/$uri /index.php;
+  root /var/www/;
```
Comment (alexAubin) [inline conf/nginx.conf]: Hmmm pretty sure the subpath is related to this ? Why use `root /var/www/` instead of the usual `alias __FINALPATH__/` ?

**Comment by alexAubin** [comment]
!testme

File: conf/nginx.conf (hunk 1 above)
Comment (Binnette) [reply conf/nginx.conf]: Hi @alexAubi, thank for your help ❤️ 
In fact I'm really not a NGINX expert. So I simply copy/paste this "root" from [Selfoss documentation](https://github.com/fossar/selfoss/wiki/nginx-configuration)...

**Comment by ericgaspar** [comment]
!testme

**Comment by Binnette** [comment]
Hi @ericgaspar, I am glad you've seen my last changes of this morning :+1: 

I finally managed to make Selfoss works with the "alias" command in Nginx instead of the "root" one. I hope it will improve integration of Selfoss in Yunohost Apps Catalog.

In short, my issue was that I didn't know how to redirect the api call of Selfoss like "/sources" to the main index.php. Because index.php contains a php router that addresses all routes/url of the app. But after a lot of Googling, I managed to write a working rewrite rule along with some location rules.

I installed successfully Selfoss on a subfolder of my yunohost server by using the branch master of this repo.

The job https://ci-apps-dev.yunohost.org/ci/job/4749 failed, so I will restart it later to see if it get stuck/timeout again. During this time, I will try other installations on my yunohost server.

Let me know if I should work again on it: do more tests, try multiples installs, do some debugging and so on.

**Pull request merged**